src/ansys/speos/core/generic/general_methods.py

```diff
@@ -283,3 +285,31 @@ def wavelength_to_rgb(wavelength: float, gamma: float = 0.8) -> [int, int, int,
     g *= 255
     b *= 255
     return [int(r), int(g), int(b), 255]
+
+
+def min_speos_version(major: int, minor: int, service_pack: int):
+    """Raise Version Warning.
+
+    Parameters
+    ----------
+    major : int
+        Major release version, e.g. 25
+    minor : int
+        Minor release version e.g. 1
+    service_pack : int
+        Service Pack version e.g. 3
+    """
+    version = f"20{major} R{minor} SP{service_pack}"
+
+    def decorator(function):
+        def wrapper(*args, **kwargs):
+            if "__init__" in function.__qualname__:
```

Comment:
```suggestion
            if "__init__" in function.__qualname__:
```
Wouldn't it be better/safer to use `.endswith("__init__")` ?